gnomad_constraint/resources/resource_utils.py

```diff
@@ -0,0 +1,62 @@
+# noqa: D100
+
+from gnomad.resources.resource_utils import (
+    TableResource,
+    VersionedTableResource,
+    DataException,
+)
+from gnomad.utils.file_utils import file_exists
+
+VERSIONS = ["2.0", "4.0"]
+CURRENT_VERSION = "2.0"
```

Comment:
We can add 4.0 when we have it, and since the dataset we will test on is 2.1.1, lets use that here

```suggestion
VERSIONS = ["2.1.1"]
CURRENT_VERSION = "2.1.1"

DATA_TYPES = ["context", "exomes", "genomes"]
GENOMIC_REGIONS = ["autosome_par", "chrx_nonpar", "chry_nonpar"]

```